### ISSUE TITLE
Cache control to delete cached pages?

### ISSUE THREAD

**Issue by pwFoo** [nicolasazrak/caddy-cache#1]
Great module!!!
Is a cache control planned to delete full cache or single cached pages? For example with a HTTP request or an admin api?

**Comment by nicolasazrak** [comment]
Yes, I have updated readme.md with that feature. I think it could be a request with a DELETE method and some header. From what I've been reading caddy will add an admin api in version 1.0, so that's another alternative.

**Comment by casdr** [comment]
Any update on this? Is this in progress already?

**Comment by nicolasazrak** [comment]
@casdr I work on this project from time to time. I will probably look into this the next time

**Comment by pwFoo** [comment]
Is cache control / delete pages from cache planned to add?

**Comment by pwFoo** [comment]
Hi @nicolasazrak,
thanks for feedback. I'm not a Go programmer, so I think I can't write a PR with needed code quality. But it's fine for now.

**Comment by lpellegr** [comment]
This would be a killer feature. Especially, if it supports surrogate keys as described in some RFC and as available with Fastly: https://docs.fastly.com/en/guides/getting-started-with-surrogate-keys
